packages/babel-parser/test/fixtures/typescript/static-blocks/invalid-static-block-with-accessibility-public/input.ts

```diff
@@ -1,3 +1,3 @@
 class Foo {
-  static public {}
```

Comment:
Q: Do we already have test cases covering `static public`?

Reply:
Yes we do. https://github.com/babel/babel/blob/1d4bd31950ec6399c9620c08b61b44fe4d5c085c/packages/babel-parser/test/fixtures/typescript/class/modifiers-invalid-order/input.ts#L21

Reply:
That one is a `static public` class method. I mean `static public {}`, which should throw. Just wondering why the test case is changed, we can always add new test cases for `public static {}`.

Reply:
Oops sorry, I misread the changed one🤦‍♂️

Reply:
Thanks, I've added more tests (5ea75b9, 828b241). However, the syntax `static` followed by modifier followed by block(`{ }`) is not recoverable. (TypeScript Compiler also doesn't seem to be able to throw a friendly error for such code ([TypeScript Playground link](https://www.typescriptlang.org/play?ts=4.4.0-beta#code/MYGwhgzhAEBiD29oG8BQ0MYgFzNglsNAA4BO+AbngKYoC+qDQA)))